osu.Game.Rulesets.Osu/osu.Game.Rulesets.Osu.csproj

```diff
@@ -4,7 +4,15 @@
     <OutputType>Library</OutputType>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Description>click the circles. to the beat.</Description>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <IncludeSymbols>true</IncludeSymbols>
```

Comment:
While it makes sense for rulesets developpers to have SourceLink enabled on the game core project (`osu.Game`), it doesn't really make any sense to enable it on rulesets projects.